client/src/main/java/org/evosuite/coverage/mutation/Mutation.java

```diff
@@ -37,7 +38,9 @@
  *
  * @author Gordon Fraser
  */
-public class Mutation implements Comparable<Mutation> {
+public class Mutation implements Comparable<Mutation>, Serializable {
```

Comment:
@fc51111, why does the `Mutation` class need to be `Serializable`?

Reply:
During the test generation process, we wanted to measure the mutation score in specific time intervals, but we kept getting null pointer exceptions. I still have an old image of this error:

![image](https://github.com/EvoSuite/evosuite/assets/58310993/5d015a38-ee26-4b7e-a098-930a04f4c8f6)

To solve this issue, it was necessary to make the "Mutation" class Serializable.